docs/architecture.md

```diff
@@ -1,103 +1,172 @@
+
+- user service
+  - `systemctl --user status ovos.service`
+- system service
+  - `systemctl status ovos.service`
+
+## ovos-core
+https://github.com/OpenVoiceOS/ovos-core
+
+This service provides the main instance for OVOS and handles all of the skill loading, and intent processing.
+
+All user queries are handled by the skills service, you can think of it as OVOS's brain
+
+typical systemd command
+
+`systemctl --user status ovos-skills`
+
+[Technical Docs on Skills](https://openvoiceos.github.io/ovos-technical-manual/skills_service/)
+
+## Messagebus
+https://github.com/OpenVoiceOS/ovos-messagebus
+
+C++ version
+
+**NOTE** This is an `alpha` version and mostly `Proof of Concept`.  It has been known to crash often
+
+https://github.com/OpenVoiceOS/ovos-bus_service
 
 The bus service provides a websocket where all internal events travel
 
 You can think of the bus service as OVOS's nervous system
 
-The mycroft-bus is considered an internal and private websocket, external clients should not connect directly to it.  **Please do not expose the messagebus to the outside world!**
+The mycroft-bus is considered an internal and private websocket, **external clients should not connect directly to it.**
+
+[Technical docs for messagebus](https://openvoiceos.github.io/ovos-technical-manual/bus_service/)
+
+typical systemd command
+
+`systemctl --user start ovos-messagebus`
+
+## Listener
+https://github.com/OpenVoiceOS/ovos-dinkum-listener
+
+The listener service is used to detect your voice.  It controlls the WakeWord, STT (Speech To Text), and VAD Plugins.  You can modify microphone settings and enable additional features under the listener section such as wake word / utterance recording / uploading
+
+The [ovos-dinkum-listener](https://github.com/OpenVoiceOS/ovos-dinkum-listener) is the new OVOS listener that replaced the origional [ovos-listener](https://github.com/OpenVoiceOS/ovos-listener) and has many more options.  Others still work, but are not recommended.
+
+[Technical Listener Docs](https://openvoiceos.github.io/ovos-technical-manual/speech_service/)
+
+typical systemd command
+
+`systemctl --user start ovos-dinkum-listener`
+
+### STT Plugins
+
+This is where speech is transcribed into text and forwarded to the skills service.
+
+Two STT plugins may be loaded at once, if the primary plugin fails for some reason the second plugin will be used.
+
+This allows you to have a lower accuracy offline model as fallback to account for internet outages, this ensures your device never becomes fully unusable.
+
+Several different STT (Speech To Text) Plugins are avaliable for use.  OVOS provides a number of public servers using the [ovos-stt-plugin-server](https://github.com/OpenVoiceOS/ovos-stt-plugin-server) plugin and are hosted by OVOS contributors.  No additional configuration is required.
```

Comment:
```suggestion
Several different STT (Speech To Text) Plugins are available for use.  OVOS provides a number of public servers using the [ovos-stt-plugin-server](https://github.com/OpenVoiceOS/ovos-stt-plugin-server) plugin and are hosted by OVOS contributors.  No additional configuration is required.
```

Reply:
Consider a blurb about how these servers are vetted, who hosts them, and general reassurance that they are as private as can reasonably be expected for something you don't host yourself.